content/technotes/2019-03-18-drake-700.Rmd

```diff
@@ -0,0 +1,169 @@
+}
+```
+
+Version 7.0.0 of [`drake`](https://github.com/ropensci/drake) just arrived on [CRAN](http://cran.r-project.org/), and it is faster and easier to use than previous releases.
+
+```{r install, eval = FALSE}
+install.packages("drake")
+```
+
+## Recap
+
+Data analysis can be slow. A round of scientific computation can take several minutes, hours, or even days to complete. After it finishes, if you update your code or data, your hard-earned results may no longer be valid. How much of that valuable output can you keep, and how much do you need to update? How much runtime must you endure all over again?
+
+For projects in R, the [`drake`](https://github.com/ropensci/drake) package can help. It [analyzes your workflow](https://ropenscilabs.github.io/drake-manual/plans.html), skips steps with up-to-date results, and orchestrates the rest with [optional distributed computing](https://ropenscilabs.github.io/drake-manual/hpc.html). At the end, [`drake`](https://github.com/ropensci/drake) provides evidence that your results match the underlying code and data, which increases your ability to trust your research.
+
+Other [pipeline tools](https://github.com/pditommaso/awesome-pipeline) such as [Make](https://www.gnu.org/software/make), [Snakemake](https://bitbucket.org/johanneskoester/snakemake/wiki/Home), [Airflow](https://github.com/apache/airflow), [Luigi](https://github.com/spotify/luigi), and [Dask](https://github.com/dask/dask) accomplish similar goals, but [`drake`](https://github.com/ropensci/drake) is uniquely suited to R. It features
+
+- A focus on ordinary R functions and variables rather than cumbersome files.
+- Heavy use of the data frame, even as a substitute for the  traditional [Makefile](https://www.gnu.org/software/make).
+- Native [tidy evaluation](https://tidyeval.tidyverse.org) support.
+- A new [domain-specific language (DSL)](http://adv-r.had.co.nz/dsl.html) for [creating large workflows](https://ropenscilabs.github.io/drake-manual/plans.html#large-plans).
+
+## Plans
+
+[`drake`](https://github.com/ropensci/drake)'s [DSL](http://adv-r.had.co.nz/dsl.html) is still experimental, but it is more powerful than old interface functions such as [`evaluate_plan()`](https://ropensci.github.io/drake/reference/evaluate_plan.html). [A new section of the manual](https://ropenscilabs.github.io/drake-manual/plans.html#large-plans) walks through the details, and the revised [gross state product example](https://ropenscilabs.github.io/drake-manual/gsp.html) is a motivating use case.
+
+Three **transformations** form the core of the new syntax.
+
+| Transformation | Tidyverse equivalent        |
+|----------------|-----------------------------|
+| `map()`        | `pmap()` from `purrr`       |
+| `cross()`      | `crossing()` from `tidyr`   |
+| `combine()`    | `summarize()` from `dplyr`  |
+
+These transformations help declare batches of multiple targets.
+
+```{r transformations}
+library(drake)
+source <- c("census", "gapminder")
```

Comment:
What type of data sources are these? Have they already been read into objects? Just a short R comment will be helpful to guide the reader.